### PR TITLE
docs: Guide > Getters

### LINKED ISSUE #5627: Doc: Guide > Getters
# Getters

## What are getters?

Describe getter functions. 

## When should I use them? 

Showing when getters are used and why they are good solution. Examples with and without getter.

## How can I use getters?

Use cases for getters.

## Available Getters 

List of all available getters divided by objects. 

## Version of Vue Storefront

- [ ] Vue Storefront
- [ ] Vue Storefront Next

## Can you complete this feature request by yourself?

- [ ] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by AdamPawlinski**
### Related Issues

closes #5627 

### Short Description and Why It's Useful

Getter docs added with examples and list of available getters.

### Which Environment This Relates To
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog
- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing Vue Storefront sites with this new feature
- [ ] (Next only) I've followed this [instruction](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and I've created a `.js` file with information about my Pull Request

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: packages/core/docs/guide/getters.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+# Getters
+
+## What are getters?
+
+Getters are pure functions which allows you to fetch response from integrated ecommerce platform. They return an agnostic or primitive type. All getters are grouped into domains they are associated with e.g. `cartGetters` and named according to values they return e.g. `getCartTotalItems`.
```
Comment (filrak) [inline packages/core/docs/guide/getters.md:5]: 1. getters are not fetching anything, they're used to 
- extract subproperties from complex objects like products and categories
- deliver agnostic data formats for the UI
2. it's worth writing this in simpler language eg "All getters are grouped into domains they are associated with" => "Each composable has its own set of getters..."

File: packages/core/docs/guide/getters.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
+# Getters
+
+## What are getters?
+
+Getters are pure functions which allows you to fetch response from integrated ecommerce platform. They return an agnostic or primitive type. All getters are grouped into domains they are associated with e.g. `cartGetters` and named according to values they return e.g. `getCartTotalItems`. 
+
+
+## How can you create and use getters?
```
Comment (filrak) [inline packages/core/docs/guide/getters.md:8]: This is for integrators. I don't think regular users need that information

Comment (AdamPawlinski) [reply packages/core/docs/guide/getters.md]: Ok removed.

File: packages/core/docs/guide/getters.md (hunk 1 above)
Comment (AdamPawlinski) [reply packages/core/docs/guide/getters.md]: I changed this part and add another quiestions to be answered in this doc.

**Pull request closed without merging**